### PR TITLE
Terrain

### PR DESCRIPTION
**Pull request by Ulaize**

### CODE REVIEW

**Comment by Ulaize** [comment]
I am unsure what it means to randomly burn and freeze tiles.

File: terrain.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+//Create constructor Tile
+
+var elementsNature = ["grass","rocks","water"];
+var conditionOfElement = ["burning", "normal","frozen"];
+
+
+function Tile (x,y){
+    this.x = x;
+    this.y = y;
+    this.height =  Math.floor(Math.random()*3);
```
Comment (ziad-saab) [inline terrain.js]: This will give a number between 0 and 2. Can you make it between 1 and 3 instead?

File: terrain.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    this.type =  elementsNature[Math.floor(Math.random()*elementsNature.length)];
+    this.condition = conditionOfElement[Math.floor(Math.random()*conditionOfElement.length)];
+    this.freeze = function (){
```
Comment (ziad-saab) [inline terrain.js]: You are missing a bunch of `else` in this function. If you put `if`s under each other, they will execute independently of each other...

File: terrain.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+            }
+        }
+    };
+    
+    
+}
+
+//Create the map 20*20
+
+function newMap (num1,num2){
```
Comment (ziad-saab) [inline terrain.js]: :+1:

File: terrain.js [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+            
+            row += x[a][b].height() +" ";
+        }
+        
+       return row;
+        
+    }
+}
+
+seeTheMap(thisIsTheMap);
```
Comment (ziad-saab) [inline terrain.js]: :+1:

File: worriors.js [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+//In addition to that, every object that gets created should have the following methods:
+//fight: will output to the console: " rushes to the arena with "
+//faceoff: faceoff takes one argument called opponent. Based on the power of each opponent, 
+//this method should output to the console which player won the fight based on their power. Be as creative as you like with the text of this method :)
+//Finally, create a bunch of warriors and make them fight together.
+
+
+Worrior.prototype = {
```
Comment (ziad-saab) [inline worriors.js]: :+1:

**Comment by ziad-saab** [comment]
The code in here works. I will close the PR, but please take a look at my comments if you want to improve your coding.

File: terrain.js (hunk 2 above)
Comment (Ulaize) [inline terrain.js]: I am unsure what you are referring to.

File: terrain.js (hunk 1 above)
Comment (Ulaize) [inline terrain.js]: I have made the changes 534b51706a3a298ffa7a52f8876c2e5f9792f46f

I see how this was not working. Thank you for the feedback :)

**Pull request closed without merging**